forms-shared/src/slovensko-sk/generateXml.ts

```diff
@@ -18,8 +20,8 @@ function getSlovenskoSkXmlBase(formDefinition: FormDefinitionSlovenskoSk, body:
   }
 }
 
-export function getEmptySlovenskoSkXml(formDefinition: FormDefinitionSlovenskoSk) {
-  const xml = getSlovenskoSkXmlBase(formDefinition, {
+export function getEmptySlovenskoSkXmlObject(formDefinition: FormDefinitionSlovenskoSk) {
```

Comment:
one more thing regarding both of these functions - they are likely good candidates to be unit-tested. If so, would do in a separate PR, just let me know what you think.

Reply:
:+1: fine, more than enough with such usage (imo), thanks for the context